<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="synps3152422" ignored="0" name="validationLogo Test" tests="2" failures="1" timestamp="2020-06-03T00:02:36 IST" time="33.304" errors="0">
  <testcase name="loginValidation" time="15.282" classname="testcases.TC01"/>
  <testcase name="logoValidation" time="20.251" classname="testcases.TC03">
    <failure type="java.lang.AssertionError" message="expected [https://www.goibibo.com/] but found [Goibibo - Best Travel Website. Book Hotels, Flights, Trains, Bus and Cabs with upto 50% off]">
      <![CDATA[java.lang.AssertionError: expected [https://www.goibibo.com/] but found [Goibibo - Best Travel Website. Book Hotels, Flights, Trains, Bus and Cabs with upto 50% off]
at testcases.TC03.logoValidation(TC03.java:33)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- logoValidation -->
</testsuite> <!-- validationLogo Test -->
